d2/src/main/java/com/linkedin/d2/balancer/clients/FailoutClient.java

```diff
@@ -90,8 +90,6 @@ public void onSuccess(URI result)
         FailoutClient.super.restRequest(redirectedRequest, requestContext, callback);
```

Comment:
nit: Can we just do `super.restRequest`. Same for line 83

Reply:
I think we cannot resolve `super` correctly in the callback.
```
Cannot resolve method 'restRequest' in 'Object'
```